ci/taskcluster/bin/decision.py

```diff
@@ -7,15 +7,23 @@
 
 tasks = [
     {
+        'name': 'recipe-client-addon:build',
+        'description': 'Build Firefox with recipe-client-addon',
+        'command': 'normandy/recipe-client-addon/bin/tc/build.sh',
+    },
+    {
         'name': 'recipe-client-addon:test',
         'description': 'Test recipe-client-addon with gecko-dev',
         'command': 'normandy/recipe-client-addon/bin/tc/test.sh',
+        'dependencies': ['recipe-client-addon:build'],
+        'artifacts_from': {
+            'BUILD_RESULT': {'task': 'recipe-client-addon:build', 'path': 'build.tar.gz'},
```

Comment:
`taskName` is a better name for this key, I think. I was confused as to why the `task` attribute could be passed to the ID maker.